libs/s3/s3_misc.py

```diff
@@ -406,5 +406,62 @@ def get_total_used(access_key: str, secret_key: str, **kwargs):
 
     size =0
     for bucket in s3_resource.buckets.all():
-        size += get_objects_size_bucket(bucket.name,access_key, secret_key, **kwargs)[1]
+        size += get_objects_size_bucket(bucket.name, access_key, secret_key, **kwargs)[1]
     return size
+
+
+def copy_object(access_key: str, secret_key: str, src_bkt: str = None, src_obj: str = None,
+                dest_bkt: str = None, dest_obj: str = None, **kwargs):
+    """
+    Copy of an object that is already stored in Seagate S3 with different permissions.
+    """
+    LOGGER.debug("Access Key : %s", access_key)
+    LOGGER.debug("Secret Key : %s", secret_key)
+    endpoint = kwargs.get("endpoint_url", S3_CFG["s3_url"])
+    LOGGER.debug("S3 Endpoint : %s", endpoint)
+
+    region = S3_CFG["region"]
+    LOGGER.debug("Region : %s", region)
+
+    s3_resource = boto3.resource('s3', verify=False,
```

Comment:
Get verify from config instead of specifying value here.

Reply:
Please understand, 
s3test lib functions are supposed to execute library functions with parameters deafult set from setup json.
s3misc is intentionally written to execute library functions with the parameter different from library function.
fro example, we want to execute access key and secret key with different than what is set in setup.json